libs/forecasting/src/lib/ForecastingDashboard/ForecastingDashboard.tsx

```diff
@@ -1,19 +1,101 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { Text } from "@fluentui/react";
+import { Dropdown, IDropdownOption, Stack, Text } from "@fluentui/react";
 import {
   defaultModelAssessmentContext,
   ModelAssessmentContext
 } from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
 import React from "react";
 
-export class ForecastingDashboard extends React.Component {
+import { ForecastComparison } from "./Controls/ForecastComparison";
+import { forecastingDashboardStyles } from "./ForecastingDashboard.styles";
+
+export class IForecastingDashboardProps {}
+
+export class IForecastingDashboardState {}
+
+export class ForecastingDashboard extends React.Component<
+  IForecastingDashboardProps,
+  IForecastingDashboardState
+> {
   public static contextType = ModelAssessmentContext;
   public context: React.ContextType<typeof ModelAssessmentContext> =
     defaultModelAssessmentContext;
 
   public render(): React.ReactNode {
-    return <Text>Placeholder</Text>;
+    const classNames = forecastingDashboardStyles();
+
+    if (
+      this.context === undefined ||
+      this.context.baseErrorCohort === undefined
+    ) {
+      return;
+    }
+
+    // "All data" cohort selected, so no particular time series selected yet.
+    // special case: only 1 time series in dataset, needs to be handled! TODO
+    const noCohortSelected =
```

Comment:
for my clarification: Will this work if user is using another language? will not be false in that case?

Reply:
The cohort name of the All data cohort will then match defaultLabel in that language, so for example, in German it will be "Alle Daten" === "Alle Daten" in case it is truly the All data cohort which in the forecasting scenario means that we haven't selected a time series. I will handle the single time series case mentioned in the comment in a follow-up PR

Reply:
... not that that should mean we have to keep doing it that way. Perhaps a flag on the cohort to indicate that it's the "all data" cohort would be useful to make this cleaner. I'm not a fan of this either... If you like this idea please upvote and I'll create a separate PR fixing all instances. If you have a better idea I'm all ears 🙂

Reply:
Are we able to check filter.length == 0?

Reply:
I like that! I'll write it as a utility function somewhere in core-ui and use it from there.

Reply:
Actually, it's more complicated. Users could create cohorts without filters that have a different name. Perhaps with injection of pre-built cohorts it's even possible to add duplicates with the same name (?)
I'm adding a utility function that does the filter comparison AND the string comparison because I'm afraid of introducing regressions in some of our older code if I remove this. Once we're more confident (e.g., by adding some additional identifier or flag indicating that it's the All data cohort, or by enforcing that there's only one cohort without filters) we can get rid of the string comparison part. 

The forecasting code doesn't care about all that and can work with just the filter-based comparison.